.yamato/upm-ci.yml

```diff
@@ -37,20 +40,20 @@ nightly_tested_releases:
       - version: 2020.3
       - version: 2021.3
       - version: 2022.2
-      - version: trunk
   - name: release_4_1
     branch: release/4.1
     nightly_editors: 
       - version: 2019.4
       - version: 2020.3
       - version: 2021.3
       - version: 2022.1
-  - name: master
-    branch: master
```

Comment:
should we keep master as well? in case this branch gets merged back to master it does not override it

Reply:
@vkovec I decide to make this PR bigger by backporting bunch of master PRs into release/5.0 branch, so that master and release/5.0 branch have the same CI setup.
Please see `List of changes` in PR description.